src/main/java/com/google/graphgeckos/dashboard/storage/DatastoreRepository.java

```diff
@@ -81,7 +81,6 @@ public boolean updateRevisionEntry(BuildBotData updateData) throws IllegalArgume
 
     if (associatedEntity != null) {
       associatedEntity.addBuilder(updateData);
-
```

Comment:
nit: [blank lines improve readability](https://softwareengineering.stackexchange.com/questions/17305/how-do-you-use-blank-lines-in-your-code)

Reply:
Fixed!